en/components/texthighlight.md

```diff
@@ -0,0 +1,340 @@
+                </button>
+            </div>
+        </igx-suffix>
+
+        <igx-suffix *ngIf="searchText.length > 0">
+            <div>
+                <span *ngIf="matchCount > 0">
+                    {{ index + 1 }} of {{ matchCount }} results
+                </span>
+                <span *ngIf="matchCount == 0">
+                    No results
+                </span>
+            </div>
+            <div class="searchButtons">
+                <button igxButton="icon" igxRipple igxRippleCentered="true" (click)="findPrev()" [disabled]="!canMoveHighlight">
+                    <igx-icon fontSet="material" name="navigate_before"></igx-icon>
+                </button>
+                <button igxButton="icon" igxRipple igxRippleCentered="true" (click)="findNext()" [disabled]="!canMoveHighlight">
+                    <igx-icon fontSet="material" name="navigate_next"></igx-icon>
+                </button>
+            </div>
+        </igx-suffix>
+    </igx-input-group>
+</div>
+```
+Then, we will add a paragraph with text and the IgxTextHighlight directive. Note that, since we need to bind the value input to the text in the paragraph, we will also use interpolation for the paragraph's text. The column, row and page inputs are usefull when you have multiple containers and can be left at 0 for our example. Another noteworthy thing is that the search container (in our case the paragraph element) needs to be the only child in its parent container and because of this we need the surrounding div element.
+
+```html
+<div>
+    <p igxTextHighlight
+        [value]="html"
+        [groupName]="'group1'"
+        [column]="0"
+        [row]="0"
+        [page]="0"
+        [containerClass]="'search-text'"
+        class="search-text">
+        {{html}}
+    </p>
+</div>
+```
+
+In the .ts file of our component first we need to add the following fields, that are used for bindings in our component's template:
+
+``` typescript
+    public html = `
+    Use the search box to search for a certain string in this text.
+    All the results will be highlighted in yellow, while the first occurrence of the string will be in orange.
+    You can use the button in the searchbox to specify if the search will be case sensitive.
+    You can move the orange highlight by either pressing the buttons on the searchbox or by using the Enter or the arrow keys on your keyboard.
+    `;
+
+    @ViewChild(IgxTextHighlightDirective, {read: IgxTextHighlightDirective})
+    public highlight: IgxTextHighlightDirective;
+
+    public searchText: string = "";
+    public matchCount: number = 0;
+    public caseSensitive: boolean = false;
+    public index: number = 0;
+
+
+    public get canMoveHighlight() {
+        return this.matchCount > 1;
+    }
+```
+
+Then we need to add the following method for allowing the use to apply the highlights for the text they have typed in the searchbox and to move the active highlight around:
+
```

Comment:
Should be 'add the following method**S**' since we are describing multiple methods.